pkg/imrpc/common.proto

```diff
@@ -8,3 +8,8 @@ enum BackendStoreDriver {
     v1 = 0;
     v2 = 1;
 }
+
+enum DataEngine {
+    engine_v1 = 0;
+    engine_v2 = 1;
+}
```

Comment:
Why not? so you don't need to do the conversion later.

```protobuf
enum DataEngine {
  v1 = 0;
  v2 = 1;
}
```

Reply:
protobuf doesn't allow the duplicate names....

Reply:
ref: https://github.com/protocolbuffers/protobuf/issues/5425

Reply:
I see, then let's follow the convention.

https://protobuf.dev/programming-guides/style/#enums

```protobuf
enum DataEngine {
  DATA_ENGINE_V1 = 0;
  DATA_ENGINE_V2 = 1;
}
```

Reply:
Updated. Thank you.